Trappist/src/Promact.Trappist.Test/Questions/QuestionsRepositoryTest.cs

```diff
@@ -357,6 +360,60 @@ public QuestionsRepositoryTest(Bootstrap bootstrap) : base(bootstrap)
         }
 
         /// <summary>
+        /// Method to test IsQuestionExistInTestAsync() method
+        /// </summary>
+        [Fact]
+        public async Task IsQuestionExistInTestTest()
+        {
+            var questionList = new List<QuestionAC>();
+            string userName = "user@domain@.com";
+            ApplicationUser user = new ApplicationUser()
+            {
+                Email = userName,
+                UserName = userName
+            };
+            await _userManager.CreateAsync(user);
+            var applicationUser = await _userManager.FindByEmailAsync(user.Email);
+            //create code-snippetQuestion
+            var codingQuestion = await CreateCodingQuestion();
+            await _questionRepository.AddCodeSnippetQuestionAsync(codingQuestion, applicationUser.Id);
+
+            var question = await _trappistDbContext.Question.FirstOrDefaultAsync(x => x.QuestionDetail == codingQuestion.Question.QuestionDetail);
+            codingQuestion.Question.Id = question.Id;
+            codingQuestion.Question.IsSelect = true;
+            questionList.Add(codingQuestion);
+            //create Test
+            var test = CreateTest("DemoTest");
+            await _testRepository.CreateTestAsync(test, applicationUser.Id);
+            await _testRepository.AddTestQuestionsAsync(questionList, test.Id);
+
+            var questionId = (await _trappistDbContext.Question.SingleAsync(x => x.QuestionDetail == codingQuestion.Question.QuestionDetail)).Id;
+            Assert.True(await _questionRepository.IsQuestionExistInTestAsync(questionId));
+        }
+
+        /// <summary>
+        /// Method to test IsQuestionExistAsync() method
+        /// </summary>
+        [Fact]
+        public async Task IsQuestionExistTest()
```

Comment:
this method and the above method is similar. why are two exact same tests written for the single method?